src/panelSocket.ts

```diff
@@ -113,4 +118,24 @@ export class PanelSocket extends EventEmitter {
 
         this.isConnected = false;
     }
+
+    private registerForCDPEvents() {
+        // register for custom events from jsdebug:
+        const registrationMessage = {
+            method: "JsDebug.subscribe",
+            params: {
+                events: [
+                "Runtime.*",
+                "DOM.*",
+                "CSS.*",
+                "DOMDebugger.*",
+                "Network.*",
+                "Page.*",
+                "Target.*",
+                "Overlay.*"
+                ]
+            }
+        }
+        this.socket?.send(JSON.stringify(registrationMessage));
```

Comment:
NIT:  I know in the DevTools we tend to avoid optional chaining with `?`.  I forget the exact reason, but it is something along the lines of it complicating and abstracting error detection.

That being said, this usage of it seems safe, but this is more of a question of if we want to establish a code-wide standard like we do with the DevTools.

Reply:
Good point. I think the standard practice in the extension so far has also been to avoid `?.`, so I'll update this usage